client/state/plugins/installed/selectors-ts.ts

```diff
@@ -95,3 +95,55 @@ export const getAllPluginsIndexedBySiteId = createSelector(
 	},
 	( state ) => [ getAllPluginsIndexedByPluginSlug( state ), getSiteIdsThatHavePlugins( state ) ]
 ) as { ( state: AppState ): { [ siteId: number ]: { [ pluginSlug: string ]: Plugin } } };
+
+export function getPluginsOnSites( state: AppState, plugins: Plugin[] ) {
```

Comment:
Is this worth some memoization?

Reply:
For now I think it's not. The heavy calculation is done in `getAllPluginsIndexedByPluginSlug` which is memoized and this selector depends on, so I believe that if this one is memoized it will have to declare the same dependents as `getAllPluginsIndexedByPluginSlug` which someone could miss updating if future changes are made to those dependents. 

In my tests on large customers I never saw this selector causing performance issues other than the ones it had from calling `getPlugins`, and memoizing introduces future maintenance concerns, so I'm inclined to not memoize it.

Reply:
Sounds good 👍